### DISCUSSION TITLE
Insert into Note at end (without specifying line #)

### DISCUSSION THREAD

**Discussion by gazoir** [mProjectsCode/obsidian-meta-bind-plugin#432] [Q&A]
Hello! I'm trying to use a button to insert some templater content into the end of the current note rather than at a specific numbered line. I wonder if there's any way to tell Metabind to insert at the end of the note? Thanks!

**Comment by lycheekitsune** [comment]
Have you tried the QuickAdd plugin? 

You can use Metabind buttons to trigger a QuickAdd command, and you should be able to use templater code to adjust the output.
![image](https://github.com/user-attachments/assets/a4972371-dfde-4985-876a-303a1e49b5b6)
